### PR TITLE
Add method to check if an element is displayed

### PR DESCRIPTION
**Pull request by aserv92**
Add method to check if an element is displayed located by it's XPath query.

### CODE REVIEW

File: src/Selenium2Driver.php [hunk 1]
```diff
@@ -802,6 +802,19 @@ public function isVisible($xpath)
...
 
+    /**
+     * Checks whether element is displayed located by it's XPath query.
+     *
+     * @param string $xpath
+     * @return Boolean
+     */
+    public function isDisplayed($xpath)
+    {
+        $element = $this->findElement($xpath);
```
Comment (Chekote) [inline src/Selenium2Driver.php:814]: Please add a `throws` to the docblock for the WebDriver\Exception thrown by findElement. Please do not use the fully qualified name in the docblock as they have done in the Selenium2Driver. Instead, use either the alias or the just the class name if `Exception` does not conflict with an existing import.

- [x] Done

File: src/Selenium2Driver.php [hunk 2]
```diff
@@ -802,6 +802,19 @@ public function isVisible($xpath)
             ($this->elementIsFullyInViewport($element) || $this->elementIsPartiallyInViewport($element));
     }
 
+    /**
+     * Checks whether element is displayed located by it's XPath query.
+     *
+     * @param string $xpath
+     * @return Boolean
```
Comment (Chekote) [inline src/Selenium2Driver.php]: PHP scalar types are always lowercase. Also, our standard is to use `bool` instead of the full word.

- [x] Done

**Review by Chekote — CHANGES_REQUESTED** [review]

File: src/Selenium2Driver.php (hunk 1 above)
Comment (aserv92) [reply src/Selenium2Driver.php:814]: Done.

File: src/Selenium2Driver.php (hunk 2 above)
Comment (aserv92) [reply src/Selenium2Driver.php]: Done.

**Review by Chekote — APPROVED** [review]

**Pull request merged**